.github/workflows/gh-actions.yml

```diff
@@ -60,12 +65,14 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: $RUNNER --test262
+      - run: $RUNNER --test262=update --build-debug
```

Comment:
ES5.1 version of test262 isn't support update yet. "update" subcommand should be added here by #4389. Or by this PR if #4389 is merged before this PR.